src/robotlibcore.py

```diff
@@ -142,6 +135,37 @@ def get_keyword_documentation(self, name):
             doc = '{}\n\n{}'.format(doc, tags) if doc else tags
         return doc
 
+    def get_keyword_types(self, keyword_name):
+        method = self.keywords.get(keyword_name)
+        if not method:
+            raise ValueError('%s is not keyword.' % keyword_name)
+        robot_types = self._get_robot_types(method)
```

Comment:
If I understand the code in `_get_robot_types` correctly, that whole method could be removed and this used instead:

```python
# Use `getattr` to support RF < 3.1 which didn't add `robot_types`.
robot_types = getattr(method, 'robot_types', ())
```

After this change the `if robot_types is not False` should naturally be removed as well.